decidim-core/app/controllers/decidim/newsletters_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Decidim
+  # The controller to handle the user's notifications dashboard.
+  class NewslettersController < Decidim::ApplicationController
+    skip_authorization_check
+
+    layout "decidim/mailer"
+    helper Decidim::SanitizeHelper
+    include Decidim::NewslettersHelper
+
+    def show
+      @organization = current_organization
+      @newsletter = collection.find(params[:id])
+      @user = current_user
```

Comment:
You should be able to use this helper method directly instead of the instance variable.